docs/hyper-params.ipynb

```diff
@@ -0,0 +1,1600 @@
+{
```

Comment:
<p>Same, suggest to call the param <code>concurrency</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/791/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>